src/Illuminate/Foundation/Application.php

```diff
@@ -454,11 +454,13 @@ public function publicPath()
     /**
      * Get the path to the storage directory.
      *
+     * @param  string  $path
      * @return string
      */
-    public function storagePath()
+    public function storagePath($path = '')
```

Comment:
If we do this change, the interface implemented "the contract" by this class needs to be changed too. And it's a breaking change, that needs to be documented.

Reply:
We can wait till 9.x to change the contract tbh?

Reply:
This is targeting already `master` @GrahamCampbell .